emmet-builders/emmet/builders/mobility_screening/migration_graph.py

```diff
@@ -0,0 +1,66 @@
+class MigrationGraphBuilder(MapBuilder):
+    def __init__(
+        self,
+        electrodes: MongoStore,
+        tasks: MongoStore,
+        migration_graphs: MongoStore,
+        ltol: float = 0.4,
+        stol: float = 0.6,
+        angle_tol: float = 15,
+        **kwargs
+    ) -> MapBuilder:
+        self.electrodes = electrodes
+        self.tasks = tasks
+        self.ltol = ltol
+        self.stol = stol
+        self.angle_tol = angle_tol
+        self.migration_graphs = migration_graphs
+        self.wi_entries = {}
+
+        super().__init__(source=electrodes, target=migration_graphs, **kwargs)
+        self.sources.append(tasks)
+
```

Comment:
Sorry if this is a repeat comment, but I think it might have gotten lost in the shuffle. Can we add a docstring for describing the builder? For example, for the StructureGroup Builder...

> """
        Aggregate materials entries into sgroups that are topotactically similar to each other.
        This is an incremental builder that makes ensures that each materials id belongs to one StructureGroupDoc document
        Args:
            materials (Store): Store of materials documents that contains the structures
            sgroups (Store): Store of grouped material ids
            query (dict): dictionary to limit materials to be analyzed ---
                            only applied to the materials when we need to group structures
                            the phase diagram is still constructed with the entire set
        """
https://github.com/materialsproject/emmet/blob/main/emmet-builders/emmet/builders/materials/electrodes.py#L93